try:
    print(10 + "4")
except Exception as e:
    print("Code error: ", e)

print("Program finished")
